WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AttributedText.swift

```diff
@@ -132,10 +132,10 @@ private struct TextViewWrapper: UIViewRepresentable {
     }
 
     func updateUIView(_ uiView: View, context: Context) {
-        uiView.attributedText = attributedText
         uiView.maxLayoutWidth = maxLayoutWidth
         uiView.font = context.environment.font?.uiFont ?? UIFont.preferredFont(forTextStyle: .body)
         uiView.textColor = context.environment.foregroundColor.map(UIColor.init)
+        uiView.attributedText = attributedText
```

Comment:
nit: any reasons for this move?

Reply:
From my testing, the font and text color set in the attributed text is overriden when setting font and text color on the label afterwards. Moving this makes sure to avoid that while keeping the font and text color set to the label in cases when no font or text color is set to the attributed text.